packages/canvas-panel-core/src/components/ThumbnailList/ThumbnailList.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withBemClass } from '@canvas-panel/core';
+//import './ThumbnailList.scss';
```

Comment:
Some default styles would be quite nice to handle the layout/sizing. On desktop with a mouse plugged in I can see scrollbars in both directions:

<img width="915" alt="screenshot 2018-12-01 at 22 11 14" src="https://user-images.githubusercontent.com/8266711/49333397-13cf3600-f5b6-11e8-8071-f7051296b6b9.png">